dfs/docker/docker.go

```diff
@@ -52,6 +52,7 @@ func IsImageNotFound(err error) bool {
 			regexp.MustCompile("(?i)could not find image"),
 			regexp.MustCompile("(?i)no such id"),
 			regexp.MustCompile("(?i)no such image"),
+                        regexp.MustCompile("(?i)manifest for .* not found"),
```

Comment:
What is this for?

Reply:
After docker was upgraded on the Jenkins node, we have got failed test here: https://github.com/control-center/serviced/blob/b990200b46df1a3b8f9791271abe6f21a0409bbe/dfs/docker/docker_test.go#L325, because of new kind of error message that isn't handled by this function https://github.com/control-center/serviced/blob/b990200b46df1a3b8f9791271abe6f21a0409bbe/dfs/docker/docker.go#L44